redis/backoff.py

```diff
@@ -1,6 +1,9 @@
 import random
 from abc import ABC, abstractmethod
 
+DEFAULT_CAP = 0.512
```

Comment:
Let's comment on why these numbers matter, and help the community contribute!

Reply:
Added